modules/server/src/main/java/org/jboss/wsf/stack/cxf/security/authentication/SubjectCreator.java

```diff
@@ -42,11 +46,16 @@
 import org.jboss.wsf.spi.security.SecurityDomainContext;
 import org.jboss.wsf.stack.cxf.security.authentication.callback.UsernameTokenCallbackHandler;
 import org.jboss.wsf.stack.cxf.security.nonce.NonceStore;
+import org.wildfly.security.auth.server.RealmIdentity;
+import org.wildfly.security.auth.server.RealmUnavailableException;
+import org.wildfly.security.auth.server.SecurityDomain;
+import org.wildfly.security.credential.PasswordCredential;
+import org.wildfly.security.password.interfaces.ClearPassword;
 
 /**
  * Creates Subject instances after having authenticated / authorized the provided
  * user against the specified SecurityDomainContext.
- * 
+ *
  * @author alessio.soldano@jboss.com
  * @author Sergey Beryozkin
  *
```

Comment:
Just BTW, I think this whole class could be refactored since legacy was removed because Subject is not being used by Elytron. However that is not part of this issue so I just plugged the support for UsernameToken here.